content/docs/concepts/csidriver/features/unity.md

```diff
@@ -493,8 +493,17 @@ allowedTopologies:
 
 This example matches all nodes where the driver has a connection to the Unity XT array with array ID mentioned via Fiber Channel. Similarly, by replacing `fc` with `iscsi` in the key checks for iSCSI connectivity with the node.
 
+**Note** : 
+  The NFS labels are automatically added by all drivers, assuming that NFS dependencies are configured by default. These dependencies come with the default Linux OS package from the node and the array supports NFS. For other protocols (iSCSI, FC, NVMe), all necessary connectivity checks between the initiator and target are performed before labels are added.
+  Since NFS labels are added by default, this process should not disrupt any other functionality, even if NFS is not configured on the array or node.
+
 You can check what labels your nodes contain by running `kubectl get nodes --show-labels` command.
 
+Here's an example of how the labels look after describing the nodes:
+csi-unity.dellemc.com/127.0.0.1--nfs=true
```

Comment:
`--` I think it will be a single` -` only after IP?

Reply:
Updated